.github/dependabot.yml

```diff
@@ -6,879 +6,1104 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+        day: "wednesday"
   - package-ecosystem: "docker"
     directory: "/"
     schedule:
       interval: "weekly"
+        day: "wednesday"
   - package-ecosystem: "gomod"
     directory: "/"
     schedule:
       interval: "weekly"
+        day: "wednesday"
   - package-ecosystem: "gomod"
     directory: "/cmd/configschema"
     schedule:
       interval: "weekly"
+        day: "wednesday"
   - package-ecosystem: "gomod"
     directory: "/cmd/mdatagen"
     schedule:
       interval: "weekly"
+        day: "wednesday"
   - package-ecosystem: "gomod"
     directory: "/cmd/otelcontribcol"
     schedule:
       interval: "weekly"
+        day: "wednesday"
   - package-ecosystem: "gomod"
     directory: "/cmd/oteltestbedcol"
     schedule:
       interval: "weekly"
+        day: "wednesday"
   - package-ecosystem: "gomod"
     directory: "/cmd/telemetrygen"
     schedule:
       interval: "weekly"
-  - package-ecosystem: "gomod"
-    directory: "/cmd/tracegen"
```

Comment:
Huh... i guess the dependabot check doesn't know to look for components that have been removed? Is that why this hasn't been an issue before?

Reply:
I believe it's just `make gendependabot` is not part of the checks CI job

Reply:
Submitted a PR https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/19405